packages/lesswrong/components/seasonal/petrovDay/PetrovWarningConsole.tsx

```diff
@@ -0,0 +1,102 @@
+// TODO: Import component in components.ts
+import React, { useEffect, useState } from 'react';
+import { registerComponent, Components } from '@/lib/vulcan-lib';
+import { gql, useQuery } from '@apollo/client';
+import { useMulti } from '@/lib/crud/withMulti';
+import { useCreate } from '@/lib/crud/withCreate';
+
+const styles = (theme: ThemeType) => ({
+  root: {
+
+  },
+  minutesRemaining: {
+    fontSize: '.9rem',
+    color: 'gray'
+  }
+});
+
+export const PetrovWarningConsole = ({classes, currentUser, side}: {
+  classes: ClassesType<typeof styles>,
+  currentUser: UsersCurrent,
+  side: 'east' | 'west'
+}) => {
+  const { PetrovWorldmapWrapper } = Components;
+
+  const { results: petrovDayActions = [], refetch: refetchPetrovDayActions } = useMulti({
+    collectionName: 'PetrovDayActions',
+    fragmentName: 'PetrovDayActionInfo',
+    terms: {
+      view: 'warningConsole',
+      side: side,
+      limit: 200
+    },
+    skip: !currentUser
+  })
+  const [lastReported, setLastReported] = useState<string | null>(null)
+  const canSendNewReport = lastReported ? Math.abs(new Date().getMinutes() - new Date(lastReported).getMinutes()) > 50 : true
```

Comment:
This logic (as well as `reportWindow`) needs to be mirrored on the server to prevent people from hitting the api directly and bypassing it